media-tv/mythtv/mythtv-31.0.ebuild

```diff
@@ -0,0 +1,431 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
```

Comment:
Any chance to get rid of python2_7 here also?

Reply:
Upstream supports python2_7 and one plugin has not been converted yet.